config/default/common/config/metadata/layers/goes/goes-west.md

```diff
@@ -1,11 +1,20 @@
-Note: This layer is generally available for the **most recent 30 days**, though certain historical ranges are also preserved.
+Note: These layers are generally available for the **most recent 30 days**, though certain historical ranges are also preserved.
+
+### GeoColor (True Color (Day), Multispectral IR (Night))
+Temporal coverage: Most recent month
```

Comment:
It may be clearer to phrase this as:
`Previous 30 days` or `Last 30 days` as the word month feels tied to calendar months.

Reply:
changed to last 30 days